ci_scripts/ci_common.sh

```diff
@@ -29,11 +29,25 @@ setup_environment () {
     bundle install --jobs 4 --retry 3
 }
 
-install_brew_dependencies () {
+install_xcode_cloud_brew_dependencies () {
     brew install "xcodegen"
     brew install "imagemagick"
 }
 
-install_python_dependencies () {
-    pip3 install -r requirements.txt # Install towncrier for generating changelogs
+install_xcode_cloud_python_dependencies () {
+    pip3 install towncrier # Install towncrier for generating changelogs
+}
+
+setup_github_actions_environment() {
+    brew install "xcodegen"
+    brew install "swiftformat"
+    brew install "git-lfs"
```

Comment:
Seems cleaner to run this in a single pass?
```suggestion
    brew install xcodegen swiftformat git-lfs
```